addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -971,10 +971,19 @@ function makeActionManager(env) {
             report_url: _getReportUrl(action, "html"),
             context: Object.assign({}, action.context),
         });
-        const clientActionOptions = Object.assign({}, options, {
-            props,
+
+        const controller = {
+            jsId: `controller_${++id}`,
+            Component: actionRegistry.get("report.client_action"),
```

Comment:
```suggestion
            // for historical reasons, the report Component is a client action,
            // but there's no need to keep this when it will be converted to owl.
            Component: actionRegistry.get("report.client_action"),
```